packages/backend/src/services/ProjectService/ProjectService.test.ts

```diff
@@ -1,4 +1,3 @@
-import { formatValue, NumberStyle } from 'common';
```

Comment:
I think you deleted these tests by mistake. Since we have other PR's making changes to these tests we can have a separate PR to move them to the common package :)

Reply:
@rephus all PR's have been merged. We can do this now.